### ISSUE TITLE
Suggestion: Add a 3D model export feature of entities

### ISSUE THREAD

**Issue by MatCat** [tsunamayo/Starship-EVO#5354]
As an avid 3D printer I would love to see a feature where one could in the ship codex click a button to export the entity to a common 3D modeling file format (stl, obj, step, etc) to be able to use for 3D printing and such.   Would be nice to have a dialog with that that would allow you to select a scale factor so that the exported object is to a desired scale.  This is a feature I wish more games would employ and imagine how much attention to the game it would get if someone sees a ship I designed in the game, 3D printed then someone asks me about it when they see it, and they learn about the game from it.

**Comment by tsunamayo** [comment]
Hi, yes it was already suggested, the problem is the shader is really custom, so people would never be able to do render in other software.

**Comment by MatCat** [comment]
The color doesn't need to be exported just the 3D model, mode 3D model formats for 3D printing don't have color anyway, though step does support coloring of faces.

**Comment by CoasterBlock** [comment]
Agreed, shader export isnt really necessary. Having the ability to export the 3D geometry (CE's included) would be really awesome!
